spec/support/standard_setup.rb

```diff
@@ -0,0 +1,161 @@
+class StandardSetup
+  include FactoryGirl::Syntax::Methods
+
+  attr_reader :cases, :user_teams
+  # rubocop:disable Metrics/MethodLength
+
+  def initialize
+    @teams = {
+       disclosure_bmt_team:       find_or_create(:team_dacu),
+       disclosure_team:           find_or_create(:team_dacu_disclosure),
+       responding_team:           create(:responding_team),
+       press_office_team:         find_or_create(:team_press_office),
+       private_office_team:       find_or_create(:team_private_office),
+       another_approving_team:    create(:approving_team)
+    }
+
+
+    @users = {
+      manager_user:                         find_or_create(:disclosure_bmt_user),
+      approver_user:                        find_or_create(:disclosure_specialist),
```

Comment:
How about using `disclosure_bmt_user` instead of `manager_user` and `disclosure_specialist` instead of `approver_user` here since we're using what we're calling the "Organisational Context" here?